common/istio-1-26/README.md

```diff
@@ -0,0 +1,97 @@
+# Istio 1.26
+
+This uses Istio 1.26 with CNI as the default configuration as described here <https://istio.io/latest/docs/setup/additional-setup/cni/>.
+
+CNI eliminates privileged init containers and improves security compliance with Pod Security Standards. This configuration also enables native sidecars for Istio through the `ENABLE_NATIVE_SIDECARS=true` environment variable in istiod.
+
+## Installation Options
+
+### Default (CNI-enabled - Recommended)
+```bash
+kubectl apply -k istio-install/base
+```
+
+### Insecure Istio (CNI-disabled)
+For environments that don't support CNI:
+```bash
+kubectl apply -k istio-install/overlays/insecure
+```
+
+### GKE-specific CNI
+GKE mounts `/opt/cni/bin` as read-only for security reasons, preventing the Istio CNI installer from writing the CNI binary. Use the GKE-specific overlay: `kubectl apply -k common/istio-1-26/istio-install/overlays/gke`. This overlay uses GKE's writable CNI directory at `/home/kubernetes/bin`. For more details, see [Istio CNI Prerequisites](https://istio.io/latest/docs/setup/additional-setup/cni/#prerequisites) and [Platform Prerequisites](https://istio.io/latest/docs/ambient/install/platform-prerequisites/)
+
+#### For Google Kubernetes Engine clusters:
+```bash
+kubectl apply -k istio-install/overlays/gke
+```
+
+### OAuth2-proxy integration
+For clusters with oauth2-proxy authentication:
+```bash
+kubectl apply -k istio-install/overlays/oauth2-proxy
+```
+
+## Switching Between Modes
+
+**Important**: You must delete the current installation before switching to avoid resource conflicts.
+
+### Switch from CNI to Non-CNI
```

Comment:
I think you can leave out the switch documentation and we shall always apply istio with the oauth2-proxy configuration